wrapper/src/main/java/software/amazon/jdbc/dialect/AuroraPgDialect.java

```diff
@@ -42,7 +42,8 @@ public class AuroraPgDialect extends PgDialect {
           + "CPU, COALESCE(REPLICA_LAG_IN_MSEC, 0), LAST_UPDATE_TIMESTAMP "
           + "FROM aurora_replica_status() "
           // filter out nodes that haven't been updated in the last 5 minutes
-          + "WHERE EXTRACT(EPOCH FROM(NOW() - LAST_UPDATE_TIMESTAMP)) <= 300 OR SESSION_ID = 'MASTER_SESSION_ID' ";
+          + "WHERE EXTRACT(EPOCH FROM(NOW() - LAST_UPDATE_TIMESTAMP)) <= 300 OR SESSION_ID = 'MASTER_SESSION_ID' "
+          + "OR LAST_UPDATE_TIMESTAMP IS NULL";
```

Comment:
can LAST_UPDATE_TIMESTAMP be NULL? should it be OK for it to be null?

what is the value of the field when a node come in?

Reply:
i was wondering the case of a cluster scaling up , to see what would happen if a node comes in and there is no LAST_UPDATE_TIMESTAMP.

overall, is it safe to consider that every time a node has this field NULL we should consider it?

Reply:
New nodes seem to have a last_update_timestamp from creation, and when nodes are deleted the entire record is removed. Even if a new node has a brief period of time where the last_update_timestamp is null, it should still be returned as part of the topology query results. It should be safe to consider all nodes with null last_update_timestamp values.